docs/global_overview.md

```diff
@@ -0,0 +1,159 @@
+# Global overview
```

Comment:
is it global overview or Deployment architecture?

Reply:
Well it's the final goal and also every diagram is generic. So maybe, generic architecture?

Reply:
Done